Package@swift-5.8.swift

```diff
@@ -0,0 +1,40 @@
+// swift-tools-version:5.8
+import PackageDescription
+
+let package = Package(
+	name: "console-kit",
+	platforms: [
+		.macOS(.v10_15),
+		.iOS(.v13),
+		.watchOS(.v6),
+		.tvOS(.v13),
+	],
+	products: [
+		.library(name: "ConsoleKit", targets: ["ConsoleKit"]),
+	],
+	dependencies: [
+		.package(url: "https://github.com/apple/swift-log.git", from: "1.5.3"),
+		.package(url: "https://github.com/apple/swift-atomics.git", from: "1.1.0")
```

Comment:
1. You didn't add this dependency to `Package.swift`, meaning the package won't build in 5.7, but more importantly,
2. You don't need this package anyway (see my comments in `ActivityBar.swift`)